sched/Kconfig

```diff
@@ -920,7 +920,13 @@ config SCHED_INSTRUMENTATION_EXTERNAL
 	bool "System performance monitor endpoints are external"
 	default n
 	---help---
-		Monitor only CPUs in the bitset.  Bit 0=CPU0, Bit1=CPU1, etc.
+		When this option isn't selected, the common code will implement
```

Comment:
This should say what will happen when it is enabled.

Reply:
Done.